iptables/iptables.go

```diff
@@ -351,7 +351,7 @@ func makeRedirectChainForOutgoingTraffic(chainName string, redirectChainName str
 }
 
 func makeShowAllRules() *exec.Cmd {
-	return exec.Command("iptables-save")
+	return exec.Command("iptables", "-t", "nat", "-L")
```

Comment:
Can you please explain the reasoning behind this change?

Reply:
Never mind, I've just say the output is much clearer this way :+1:

Reply:
running that command looks like was hard to do without root. and this command still meets the intended objective.